languages/python/config.json

```
@@ -20,14 +20,20 @@
         "uuid": "7ecd42c1-2ed5-487a-bc89-71c9cbad9b05",
         "concepts": ["tuples"],
         "prerequisites": [
-          "boleans",
+          "bools",
           "for-loops",
           "functions",
           "if-keyword",
           "in-keyword",
           "integers",
           "return-keyword"
         ]
+      },
+      {
+        "slug": "bools",
+        "uuid": "c050470d-ef22-4187-aaac-239e07955a4d",
+        "concepts": ["bool"],
```

Should this be:

```suggestion
        "concepts": ["bools"],
```

The prerequisite above also uses `bools`, and we usually use plural instead of singular.

---

I think I was following the design issue, but I agree and can make this change consistent.